exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt

```diff
@@ -100,10 +105,11 @@ infix fun Expression<Boolean>.and(op: Expression<Boolean>): Op<Boolean> = when {
     this is AndOp && op is AndOp -> AndOp(expressions + op.expressions)
     this is AndOp -> AndOp(expressions + op)
     op is AndOp -> AndOp(
-        ArrayList<Expression<Boolean>>(op.expressions.size + 1).also {
-            it.add(this)
-            it.addAll(op.expressions)
-        }
+        ArrayList<Expression<Boolean>>(op.expressions.size + 1)
+            .also {
+                it.add(this)
+                it.addAll(op.expressions)
+            }
```

Comment:
Which detekt rule is this based on (and the change above)? Is it something set in the project .yml or based on default detekt settings?

Reply:
This came from Reformat Code (⌥⌘L), but I don't think it's necessary actually, so I'll remove it.

Reply:
Correction: This did not come from Reformat Code. I must have done that before or after Reformat Code and forgot about it.